docs/SRS-Volere/SRS.tex

```diff
@@ -171,15 +171,27 @@ \subsection{Longevity Requirements}
 
 \section{Operational and Environmental Requirements}
 \subsection{Expected Physical Environment}
-\lips
+\begin{enumerate} [{OE-PE}1.]
+  \item Application should be functional in City of Hamilton, Water Division sites and offices.
+\end{enumerate}
 \subsection{Wider Environment Requirements}
-\lips
+\begin{enumerate} [{OE-WE}1.]
+  \item Application should be functional on Mobile and Desktop web browser environments.
+  \item Application should be able to run on any device with a modern web browser.
+\end{enumerate}
 \subsection{Requirements for Interfacing with Adjacent Systems}
-\lips
+\begin{enumerate} [{OE-IAS}1.]
+  \item Application should integrate with existing SharePoint repositories.
```

Comment:
should also describe infor and MySDS integrations

Reply:
Added requirements for EAM and MySDS integration.